bundles/automation/org.eclipse.smarthome.automation.json.parsers/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <groupId>org.eclipse.smarthome.bundles</groupId>
+    <artifactId>automation</artifactId>
+    <version>0.8.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <bundle.symbolicName>org.eclipse.smarthome.automation.json.parsers</bundle.symbolicName>
+    <bundle.namespace>org.eclipse.smarthome.automation.json.parsers</bundle.namespace>
+  </properties>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.eclipse.smarthome.automation</groupId>
+  <artifactId>org.eclipse.smarthome.automation.json.parsers</artifactId>
+
+  <name>Eclipse SmartHome Automation JSON Parsers</name>
+
+  <packaging>eclipse-plugin</packaging>
+
+	<dependencies>
+		<dependency>
```

Comment:
Instead of doing a Maven dependency, org.json should be added to the smarthome.target instead. 
Note that it is available on Orbit at http://download.eclipse.org/tools/orbit/downloads/drops/R20140525021250/repository/plugins/org.json_1.0.0.v201011060100.jar, so adding it to the target platform should be very simple.